pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -72,6 +73,10 @@ const (
 var (
 	// ErrVersionNotSupported is returned when the api version of runtime interface is not supported
 	ErrVersionNotSupported = errors.New("runtime api version is not supported")
+	// ignore error regexex for sandbox creation
+	ignoreSandboxCreateErrorRegex = []*regexp.Regexp{
```

Comment:
As a follow up can we make the CRI and container runtimes return structured errors?  @mrunalp and I had discussed it, but I didn't know whether it landed (GRPC gives us the tools to have the runtimes do the right thing and give us a typed error).

Reply:
Yes. +1